src/util/bip32.rs

```diff
@@ -849,7 +878,7 @@ mod tests {
                  expected_pk: &str) {
 
         let mut sk = ExtendedPrivKey::new_master(network, seed).unwrap();
-        let mut pk = ExtendedPubKey::from_private(secp, &sk);
+        let mut pk = ExtendedPubKey::from_priv(secp, &sk);
```

Comment:
Is this used so often that removing three letters is worth readability and larger diff?

Reply:
It was not removed for readability, but for standardization. Most of method names uses `pub`/`priv` or `secret`, not `private`